### DISCUSSION TITLE
Sumatra failing to uninstall

### DISCUSSION THREAD

**Discussion by lcunn** [sumatrapdfreader/sumatrapdf#3205] [Documentation]
I am trying to uninstall Sumatra PDF. I installed it on Windows 10 using the official installer. I go to the uninstall apps section in windows settings, click uninstall, and enter the admin password. Then a window titled SumatraPDF 3.4.6 Uninstaller comes up, saying 'Are you sure you want to uninstall SumatraPDF?' Then I click uninstall SumatraPDF. Then it says it has been uninstalled. But SumatraPDF is still an app on my device, which I can open and use. I have tried going through my admin account, but then app is not installed there.

What do I do?

**Comment by GitHubRulesOK** [comment]
hmm ocassionally with using multiple versions I get blocked even as admin, so it depends where the files are  and most importantly if one or both of the two dlls is active, since search and or thumbnail/preview are locked into the system.
To stop search service you may need to temporarily kill via task manager, for preview temporarily switch explorer off as well, those are only needed if you are confident one or both are the cause of uninstaller blocking by keeping SumatraPDF dlls locked as active.
